pkg/acsengine/defaults.go

```diff
@@ -313,6 +313,24 @@ var (
 			},
 		},
 	}
+
+	// DefaultContainerMonitoringAddonsConfig is the default container monitoring Kubernetes addon Config
+	DefaultContainerMonitoringAddonsConfig = api.KubernetesAddon{
+		Name:    ContainerMonitoringAddonName,
+		Enabled: helpers.PointerToBool(api.DefaultContainerMonitoringAddonEnabled),
+		Config: map[string]string{
+			"omsAgentVersion":       "1.6.0-42",
+			"dockerProviderVersion": "2.0.0-2",
+			"workspaceGuid":         "c5905df1-b3b9-42b9-acf0-14a4c4ef028c",
```

Comment:
@rashmichandrashekar What is the value of having a default workspaceGuid and workspaceKey? Don't these always have to be user-provided?

Reply:
They do.. But for testing purposes we want to use this workspace, since the functionality we are testing doesn't involve validating data in the workspace.